logs_ingest/dynatrace_client.py

```diff
@@ -19,6 +19,7 @@
 import time
 import aiohttp
 import asyncio
+import ast
```

Comment:
so how does this work? Is there a way to avoid using ast library?

Reply:
Well the ast is used to convert a string looking as list to list. The ast.literal_eval is created specifically for that kind of operations. Of course, I can walk-around this library but other solutions might break is some corner cases.

Reply:
Well I use ast.literal_eval to convert a string looking as list to a list. Of course, I can walk-around this library but other solutions might break is some corner cases.

Reply:
Leave it like that, since it built into python it's fine for me